activejob/test/cases/callbacks_test.rb

```diff
@@ -48,7 +51,9 @@ class CallbacksTest < ActiveSupport::TestCase
     ActiveJob::Base.skip_after_callbacks_if_terminated = true
     reload_job
     job = AbortBeforeEnqueueJob.new
-    job.enqueue
+    ActiveSupport::Deprecation.silence do
```

Comment:
Similar comment as above but inversed this time. The triggered deprecation is something unrelated to the change here